### ISSUE TITLE
Running ApprovalTests in different OS

### ISSUE THREAD

**Issue by aaltat** [approvals/ApprovalTests.Python#41]
When I run my ApprovalTests in Linux, the line feed is written in `LF`. Now if I run same tests in Windows tests fail, because the line feed is different, in windows: `CRLF`. Is there a way to configure ApprovalTests to ignore the linefeed or something which would allow the execution of the test in different OS?

**Comment by objarni** [comment]
@aaltat the result of the function you are testing is in your control; if it outputs LF on one platform and CRLF on another, there is a problem with the string representation of the return value of your function - not ApprovalTests.Python.

**Comment by aaltat** [comment]
Actually my function is not outputting linefeed at all. Instead ApprovalTests.Python is adding line feed, in the [list_utils.py](https://github.com/approvals/ApprovalTests.Python/blob/master/approvaltests/list_utils.py#L6). If I do this: `verify_all('header', ['result 1', 'result 2`])` then the approved file looks like this:
``` 
header

0) result 1
1) result 2
```
And because line feeds are added, for formatting reasons (which I totally understand and agree), the line feed causes a problem when tests are run in OS using different line feed. Therefore it would be nice to have functionality which allows to hard code the line feed (which may be difficult in Python) or an way to ignore the line feed.

**Comment by objarni** [comment]
Ah, I see. I'm new to ApprovalTests.Python and have only ever used the combinatorics verification: verify_all_combinations(fn, args).

In that use case fn is defined by myself and I'm in control of the output.

One idea to get your use case cross-platform would be to wrap the parameters into a local function:

```
   def test_by_verfication(self):
      def fn(a, b):
         return "your platform independent formatting string".format(a, b)
      verify_all(fn('header', ['result 1', 'result 2']))
```

**Comment by rob-smallshire** [comment]
Actually, the default ApprovalTests.Python implementation opens file for write in text mode in an essentially arbitrary encoding, with no means to control either the encoding or newline translation behaviour. See #50 for more details. I have a PR in the works which offers improved control over newline translation behaviour.

**Comment by aaltat** [comment]
OK, I see your point and that would be useful enhancement. Should we close this issue, because it is quite similar?

**Comment by isidore** [comment]
This should be fixed in version 0.3.
